roxctl/common/npguardutils/npgLogger.go

```diff
@@ -10,7 +10,8 @@ type npgLogger struct {
 	l logger.Logger
 }
 
-func newNpgLogger(l logger.Logger) *npgLogger {
+// NewNpgLogger creates a new npgLogger
+func NewNpgLogger(l logger.Logger) *npgLogger {
```

Comment:
`npg` is repeated in the name of package and the function. We need to rethink the naming

Reply:
How the following idea would sound?

File: `roxctl/common/npg/logger.go`
Content (fragments): 

```go
package npg

type Logger struct {...}

func NewLogger(l logger.Logger) *Logger {...}
```

Reply:
done, 
however, the struct `type Logger struct {...}` should be unexported.
called it `log` since `logger` caused compiling issue. ( another option may be `nlogger` !)

Reply:
Thanks for moving it, it looks much better now.

Why do you think that the struct should be unexported? This type is returned by `NewLogger`, so other pkgs will use it. Using unexported type is really annoying, so I would suggest to export it or provide a reason for not exporting it (my IDE suggests to export it - see screenshot).

![Screenshot 2023-04-25 at 15 41 38](https://user-images.githubusercontent.com/114479/234295648-45cd9d79-54f5-4359-872b-4ebb61a0d9c3.png)

Reply:
I see, exported